src/tests/test_bse.cc

```diff
@@ -1,864 +1,871 @@
-/*

```

Comment:
why delete this?

Reply:
I have not deleted the tests, I just run `dos2unix` because I am having formatting issues, you can check the file and see that the code is down there.